base/message/example/identity-and-trust-protocol-context.json

```diff
@@ -0,0 +1,4 @@
+{
+    "@context": "https://www.w3.org/ns/did/v1",
+    "version": "1.0"
```

Comment:
```suggestion
```
we probably don't needa `version` or `@version` property in/near the `@context`. It's optional anyway and I don't think we'll use any of the features definted between json-ld 1.0 and 1.1.

Reply:
done